draft-ietf-modpod-group-processes.md

```diff
@@ -11,7 +11,7 @@ v: 3
 venue:
     mail: mod-discuss@ietf.org
     github: larseggert/moderation
-    latest: https://larseggert.github.io/moderation/draft-ietf-modpod-group-processes.html
+    latest: https://github.com/larseggert/draft-ietf-modpod-group-processes
```

Comment:
```suggestion
    https://larseggert.github.io/draft-ietf-modpod-group-processes/draft-ietf-modpod-group-processes.html
```
You're correct that the old link is wrong, but I think your new link is also wrong, because it goes to the repo, not the editors' copy.